pkg/doc/did/contexts.go

```diff
@@ -1,294 +0,0 @@
-/*
```

Comment:
why was this deleted and then moved to .jsonld files? Don't we need these contexts in production?

Reply:
Right, it was moved to the `*.jsonld` files of test data directory. 
I have no strong opinion on whether we should have the cached contexts (by default) in the production.
What do you think @troyronda ?

Reply:
I think the default should be cached. The question now is where to maintain them, in files or in go code....

Go code is more straightforward imo.

Reply:
I agree we need them in Go code

Reply:
Yes, this makes sense. I added back the caching to the production code. It's also exported now: `CachingJSONLDLoader()`. This allows the users to use the default caching or apply their own via providing custom loader as `jsonld.ProcessorOpts`.